protocol/x/ratelimit/keeper/keeper.go

```diff
@@ -122,17 +128,13 @@ func (k Keeper) ProcessDeposit(
 //	baseline = max(baseline_minimum, baseline_tvl_ppm * current_tvl)
 func (k Keeper) GetBaseline(
 	ctx sdk.Context,
-	denom string,
+	currentSupply *big.Int,
```

Comment:
Change function signature, because whereever `GetBaseline` is called for a limiter, that caller function could've called `bankKeeper.GetSupply()` for all limiters of a denom.